src/hb-font.hh

```diff
@@ -478,8 +478,8 @@ struct hb_font_t
     hb_memset (extents, 0, sizeof (*extents));
 
     /* This one is messy gonna be messy. */
-    if (synthetic && !is_synthetic ())
-      synthetic = false;
+    //if (synthetic && !is_synthetic ())
+    //  synthetic = false;
```

Comment:
Did you mean to keep the commented out code and the (now outdated/misplaced) comment?

Reply:
Fixed. Thanks.